js/myparcel.js

```diff
@@ -21,7 +21,7 @@ MyParcel = {
         $('#mypa-morning-delivery').html(' ( + &euro;' + this.data.config.priceMorningDelivery +')');
         $('#mypa-evening-delivery').html(' ( + &euro;' + this.data.config.priceEveningDelivery +')');
 		$('#mypa-signature-price').html(' ( + &euro;' + this.data.config.priceSignature +')');
-        $('#mypa-only-recipient').html(' ( + &euro;' + this.data.config.priceOnlyRecipient +')');
+        $('#mypa-only-recipient-price').html(' ( + &euro;' + this.data.config.priceOnlyRecipient +')');
```

Comment:
Misschien beter om alles met prijzen; `' ( + &euro;' + this.data.config.priceOnlyRecipient +')'` te vervangen met `getPriceHtml(this.data.config.priceOnlyRecipient)`. En dat je dan in die (nieuwe) functie die euroteken toevoegt (of gratis).

Reply:
Done!